package.json

```diff
@@ -16,7 +16,8 @@
     "packages/tests",
     "packages/browser-tests",
     "packages/build-utils",
-    "packages/react-native-polyfills"
+    "packages/react-native-polyfills",
+    "packages/scalable-data-sync"
```

Comment:
how about `packages/sds`? 
following how `nim-sds` is named

Reply:
My concern is that `sds` is very close to `sdk`...

Reply:
valid concern - but I wouldn't pay much attention to that - `s` and `k` are quite far from each other on `qwerty` keyboard usually

Reply:
updated package name to `sds`